src/chess.ts

```diff
@@ -1961,6 +1981,8 @@ export class Chess {
       output = 'O-O'
     } else if (move.flags & BITS.QSIDE_CASTLE) {
       output = 'O-O-O'
+    } else if (move.flags & BITS.NULL_MOVE){
+      output = SAN_NULLMOVE
```

Comment:
Could you just return SAN_NULLMOVE here to skip the rest of this function?

Reply:
Good point, I fixed this.